src/calendar/google_calendar/mod.rs

```diff
@@ -0,0 +1,66 @@
+use crate::clients::google::calendar::*;
+use crate::clients::google::token_storage::TokenStorage;
+use crate::clients::google::GoogleClient;
+
+use chrono::prelude::*;
+use std::convert::From;
+use std::error;
+use time::Duration;
+
+use super::{CalendarFacade, Event};
+
+#[derive(Debug, Clone)]
+pub struct Calendar<T: TokenStorage> {
+    calendars: Vec<String>,
+    google_client: GoogleClient<T>,
+}
+
+impl<T: TokenStorage> Calendar<T> {
+    pub fn new(calendars: Vec<String>, google_client: GoogleClient<T>) -> Self {
+        Self {
+            calendars,
+            google_client,
+        }
+    }
+}
+
+impl<T: TokenStorage> CalendarFacade for Calendar<T> {
+    fn get_events(&self) -> Result<Vec<Event>, Box<error::Error>> {
+        let start = dbg!(Utc::now());
+        let end = dbg!(start + Duration::weeks(1));
```

Comment:
ale to tak zostawiasz `dbg!` w produkcyjnym kodzie?

Reply:
Tak, jak kompilujesz z `--release` to to makro jest noopowane.

Reply:
Nice!